packages/itwin/measure-tools/src/measurements/AreaMeasurement.ts

```diff
@@ -42,6 +42,7 @@ import { MeasureTools } from "../MeasureTools";
  */
 export interface AreaMeasurementProps extends MeasurementProps {
   polygonPoints: XYZProps[];
+  ratio?: number;
```

Comment:
Please use a name that is more meaningful. ratio can go either side (do I multiply or divide by ratio??)